deploy/rinkeby.ts

```diff
@@ -1,41 +1,104 @@
+  const reserveResult = await deployAndLog('Reserve', { from: deployer, args: [deployer, ticketResult.address] })
+  await deployAndLog('DrawCalculatorTimelock', { from: deployer, args: [deployer, drawCalculatorResult.address] })
+  await deployAndLog('EIP2612PermitAndDeposit', { from: deployer })
+  
+  // New Draw Every 4 Hours
+  const calculatedBeaconPeriodSeconds = 86400 / 6;
+
+  const drawBeaconResult = await deployAndLog('DrawBeacon', {from: deployer, args: [
+    deployer,
+    drawBufferResult.address,
+    rngServiceResult.address,
+    1,
+    parseInt('' + ((new Date().getTime() / 1000) - calculatedBeaconPeriodSeconds)),
+    calculatedBeaconPeriodSeconds,
+    RNG_TIMEOUT_SECONDS
+]});
+
+  const prizeDistributionFactoryResult = await deployAndLog('PrizeDistributionFactory', {
+    from: deployer,
+    args: [
+      deployer,
+      prizeTierHistoryResult.address,
+      drawBufferResult.address,
+      prizeDistributionBufferResult.address,
+      ticketResult.address,
+      PRIZE_DISTRIBUTION_FACTORY_MINIMUM_PICK_COST // 1 USDC
+    ]
+  })
+
+  const prizeFlushResult = await deployAndLog('PrizeFlush', { from: deployer, args: [deployer, prizeDistributorResult.address, prizeSplitStrategyResult.address, reserveResult.address]})
+  const beaconTimelockTriggerResult = await deployAndLog('BeaconTimelockTrigger', { from: deployer, args: [deployer, prizeDistributionFactoryResult.address, drawCalculatorResult.address]})
+
+  // ===================================================
+  // Configure Contracts
+  // ===================================================
+  await pushDraw1()
+  await initPrizeSplit()
+  await setTicket(ticketResult.address)
+  await setPrizeStrategy(prizeSplitStrategyResult.address)
+  await setManager('BeaconTimelockTrigger', null, defenderRelayer)
+  await setManager('DrawBuffer', null, drawBeaconResult.address)
+  await setManager('PrizeFlush', null, defenderRelayer)
+  await setManager('Reserve', null, prizeFlushResult.address)
+  await setManager('DrawCalculatorTimelock', null, beaconTimelockTriggerResult.address)
+  await setManager('PrizeDistributionFactory', null, beaconTimelockTriggerResult.address)
+  await setManager('PrizeDistributionBuffer', null, prizeDistributionFactoryResult.address)
+
+  await transferOwnership('PrizeDistributionFactory', null, executiveTeam)
+  await transferOwnership('DrawCalculatorTimelock', null, executiveTeam)
+  await transferOwnership('PrizeFlush', null, executiveTeam)
+  await transferOwnership('Reserve', null, executiveTeam)
+  await transferOwnership('YieldSourcePrizePool', null, executiveTeam)
+  await transferOwnership('PrizeTierHistory', null, executiveTeam)
+  await transferOwnership('PrizeSplitStrategy', null, executiveTeam)
+  await transferOwnership('DrawBuffer', null, executiveTeam)
+  await transferOwnership('PrizeDistributionBuffer', null, executiveTeam)
+  await transferOwnership('BeaconTimelockTrigger', null, executiveTeam)
```

Comment:
Let's use `incOperations` for all ownership on testnets.  The exec team doesn't exist!